src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -381,12 +394,11 @@ func applyClientConfig(client *client.Client, cfg *config.Config) {
 
 	// We offload this onto a go routine because creating the http clients
 	// blocks on waiting for the local server, and the local server starts
-	// on the thread, otherwise creating a deadlock.
+	// later on this same thread, so it would otherwise creating a deadlock.
 	go func() {
-		withHttpClient(cfg.Addr, config.Configure)
```

Comment:
This is essentially replaced with the StartPolling call. We also don't want to call this on every config change.